src/map/unit.c

```diff
@@ -2836,6 +2836,7 @@ static int unit_free(struct block_list *bl, enum clr_type clrtype)
 		case BL_PC:
 		{
 			struct map_session_data *sd = BL_UCAST(BL_PC, bl);
+			int i = 0;
```

Comment:
better left ``i`` in place where it was before
variables should be smaller scope as possible